matrix_is_tester/test_account.py

```diff
@@ -18,9 +18,9 @@
 
 import unittest
 
-from .fakehs import get_shared_fake_hs, token_for_user
-from .is_api import IsApi
-from .launch_is import get_or_launch_is
+from fakehs import get_shared_fake_hs, token_for_user
```

Comment:
shouldn't this be `from matrix_is_tester.fakehs` ?

I get:

```
Traceback (most recent call last):
  File "/home/rav/work/matrix-is-tester/env/lib/python3.6/site-packages/twisted/trial/runner.py", line 531, in loadPackage
    module = modinfo.load()
  File "/home/rav/work/matrix-is-tester/env/lib/python3.6/site-packages/twisted/python/modules.py", line 392, in load
    return self.pathEntry.pythonPath.moduleLoader(self.name)
  File "/home/rav/work/matrix-is-tester/env/lib/python3.6/site-packages/twisted/python/reflect.py", line 308, in namedAny
    topLevelPackage = _importAndCheckStack(trialname)
  File "/home/rav/work/matrix-is-tester/env/lib/python3.6/site-packages/twisted/python/reflect.py", line 255, in _importAndCheckStack
    reraise(excValue, excTraceback)
  File "/home/rav/work/matrix-is-tester/matrix_is_tester/test_account.py", line 21, in <module>
    from fakehs import get_shared_fake_hs, token_for_user
builtins.ModuleNotFoundError: No module named 'fakehs'
```

Reply:
Mmm, I think I still don't understand what best practice is with python imports / how this was working for me, but yes, I guess this makes sense (done it for all the other imports too).